internal/datanode/importv2/util.go

```diff
@@ -202,6 +207,49 @@ func AppendSystemFieldsData(task *ImportTask, data *storage.InsertData) error {
 	return nil
 }
 
+func RunEmbeddingFunction(task *ImportTask, data *storage.InsertData) error {
```

Comment:
`task Task` is better, and then getting schema by `task.GetSchema()`

Reply:
why `task Task` is better? I was following the signature of `AppendSystemFieldsData` method right above this one.